Gems/ROS2/Code/Include/ROS2/Frame/ROS2FrameComponent.h

```diff
@@ -50,6 +50,10 @@ namespace ROS2
         //! Set a above-mentioned frame id
         void SetFrameID(const AZStd::string& frameId);
 
+        //! Set is Frame is dynamic or static
+        //! @param isDynamic true for dynamic frame, that should be published to `/tf`
+        void SetFrameDynamic(const bool isDynamic);
```

Comment:
Quick look: it seems that we only want to set this if there are joints (other than Fixed joint) attached. Is there any need to expose this to the user?